keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -691,11 +755,18 @@ defined in the k8s.io/apis/core/v1 package. Thus, every addition of a new
 condition type will require an API review. The constants will allow users of the
 package to reduce the risk of typing mistakes.
 
-Additionally, for Beta, we will re-evaluate an idea of a generic opinionated condition
-type indicating that a pod can be retried, for example `DisruptionTarget`.
+Additionally, we introduce a pair of generic condition types:
+- `DisruptionTarget`: indicates pod disruption (due to e.g. preemption,
```

Comment:
just `Disrupted` might be sufficient and it is analogous to `ResourceLimitExceeded` in describing what happened to the pod.

Reply:
Our motivation to call it `DisruptionTarget` was that the actual Pod deletion may not happen. For example, kube-scheduler wants to preempt a pod so it adds the DisruptionTarget condition. However, the actual deletion may fail and another Pod might be chosen the next time. In order to clean up the potential stale DisruptionTarget conditions we have added the disruption worker: https://github.com/kubernetes/kubernetes/pull/111475.

On the other hand the fact that the operation is performed in two steps seems to be a leaking implementation detail. For example, if there is an API for atomic Pod status update and deletion then we could do it in one step. Then `Disrupted` would be the preferred name. @alculquicondor  any opinion?

Reply:
Ok, let's stay with `DisruptedTarget` unless some new suggestion or argument.

Reply:
Sounds good if that is the case, although I expect the name to leave some users a bit puzzled